android/src/main/java/io/dcvz/rnzendesk/RNZendeskBridge.java

```diff
@@ -235,4 +237,33 @@ public void onError(ErrorResponse errorResponse) {
         }
   
     }
+
+    @ReactMethod
+    public void getRequests(final String statuses, final Promise promise) {
+        final RequestProvider provider = Support.INSTANCE.provider().requestProvider();
+        final WritableArray transformedRequests = new WritableNativeArray();
+
+        provider.getRequests(statuses, new ZendeskCallback<List<Request>>() {
+            @Override
+            public void onSuccess(List<Request> requests) {
+                for (Request r : requests) {
+                    WritableMap request = new WritableNativeMap();
+                    request.putString("id", r.getId());
+                    request.putString("status", r.getStatus().name().toLowerCase());
+                    request.putString("subject", r.getSubject());
+                    request.putString("lastComment", r.getLastComment().getBody());
+                    request.putString("updatedAt", Long.toString(r.getUpdatedAt().getTime()));
```

Comment:
Can we get tags?

Reply:
We can do, but that can be added easily later - the tags aren't required for the bit of app code these changes are for